src/store/hotspotDetails/hotspotDetailsSlice.ts

```diff
@@ -0,0 +1,97 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit'
+import {
+  HotspotRewardData,
+  HotspotRewardSumData,
+} from '@helium/http/build/models/HotspotReward'
+import { Hotspot } from '@helium/http'
```

Comment:
I think all of these types can be imported from `@helium/http`
`import { Hotspot, HotspotReward, HotspotRewardSum } from '@helium/http'`

Reply:
Fixed